ncc/client/dummy.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2012 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Author: jsing@google.com (Joel Sing)
+
+package client
+
+// This file contains dummy implementation of interfaces for testing purposes
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/google/seesaw/common/seesaw"
+	"github.com/google/seesaw/ipvs"
+	ncctypes "github.com/google/seesaw/ncc/types"
+	"github.com/google/seesaw/quagga"
+
+	log "github.com/golang/glog"
+)
+
+type dummyNCC struct{}
+
+func NewDummyNCC() *dummyNCC {
```

Comment:
could you add document for NewDummyNCC() and NewLBInterface() and make it clear it's for testing purpose?

Reply:
Done